### PR TITLE
Add Axios

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: img/a/axios.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64"><path fill="#fff" d="M55.4 22.08h-7.225l-4.25 7.055-4.122-7.055h-7.3l7.947 11.645-5.142 7.522-8.457-19.125h-6.375l10.54 23.714h7.947l4.972-7.522 4.93 7.522h6.502l-7.862-12.112z"/><path fill="#0a99e0" d="M14.82 59.725H0L24.47 4.223H39.4z"/><path fill="#222" d="M49.137 59.725 64 59.778 39.4 4.222H24.47l24.668 55.502"/></svg>
```
Comment (Carlgo11) [inline img/a/axios.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><path fill="#fff" d="M55.4 22h-7.2l-4.3 7.1-4.1-7h-7.3l8 11.6-5.2 7.5-8.4-19h-6.4L31 45.7h8l5-7.5 4.9 7.5h6.5l-7.9-12z"/><path fill="#0a99e0" d="M14.8 59.7H0L24.5 4.2h14.9z"/><path fill="#222" d="M49.1 59.7H64L39.4 4.3h-15l24.7 55.5"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
What site criteria make it qualify to be listed on 2FA Directory?

**Comment by soodaayush** [comment]
It would meet Personal Info, considering you can store your full name there.

**Comment by Carlgo11** [comment]
With an emphesis on <b><u>can</u></b> as in is optional. A name is already a rather weak PII by itself as it's often public information. It being the sole PII means theres even less of a reason to list it.

Is there any other sensitive information like payment information or does the account give access to otherwise inaccessible articles?

**Comment by soodaayush** [comment]
It can store payment information because you can subscribe to their service called Axios Pro, where it apparently provides exclusive news and analysis at the price of $600 every year.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**